### DISCUSSION TITLE
Feature Request: Volume Consistent Transform Option

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#242] [Ideas]
I'd like a transform option that keeps the volume of the object your transforming consistent, something that's like freeform transformation, but as you squash the Y-Scale, the X-Scale stretches to keep the volume consistent, which is useful for bouncy ball animations.

**Comment by DinoAwakening** [comment]
You can just do exactly the thing you’re explaining with the Freeform transform. Or did I misunderstood something? :v
